src/components/TodoList.js

```diff
@@ -45,13 +45,21 @@ function TodoList() {
             return todo;
         })
         setTodos(updatedTodos);
+        console.log(todos)
```

Comment:
Remove all the console log's from this file.

Reply:
Done